### PR TITLE
Fixing minor issues

### PR DESCRIPTION
**Pull request by sp-daniel-ramos**
Attending @gonzaloserrano comments on https://github.com/socialpoint-labs/bsk/pull/41

### CODE REVIEW

File: httpx/client_test.go [hunk 1]
```diff
@@ -53,29 +53,32 @@ func (mi *MetricsInstrument) Timer(name string, tags ...metrics.Tag) metrics.Tim
 	return mi.timerMock
 }
 
-type TimerMock struct {
+type TimerSpy struct {
 	metrics.Timer
-	TagsMap map[string]interface{}
-	Started bool
-	Stopped bool
+	TagMapMutex sync.RWMutex
```
Comment (gonzaloserrano) [inline httpx/client_test.go]: the mutex should be unexported and used ofnly inside the TimerSpy methods. 

usually is called `mux` and put above the fields that it protects, with an empty line above it:
```go
type TimerSpy struct {
    metrics.Timer

    mu sync.RWMutex
    TagsMap ...
}
```

Comment (sp-daniel-ramos) [reply httpx/client_test.go]: then I would have to create reading functions in order to make it concurrent?

Comment (sp-ludovic-ivain) [reply httpx/client_test.go]: yes

File: httpx/client_test.go [hunk 2]
```diff
@@ -53,29 +53,32 @@ func (mi *MetricsInstrument) Timer(name string, tags ...metrics.Tag) metrics.Tim
...
+	TagsMap     map[string]interface{}
+	Started     bool
+	Stopped     bool
 }
 
-func (tm *TimerMock) WithTags(tags ...metrics.Tag) metrics.Timer {
+func (tm *TimerSpy) WithTags(tags ...metrics.Tag) metrics.Timer {
 	for _, t := range tags {
 		tm.TagsMap[t.Key] = t.Value
```
Comment (sp-ludovic-ivain) [inline httpx/client_test.go]: don't you need to lock also here ?

Comment (sp-daniel-ramos) [reply httpx/client_test.go]: missed that one, done.

File: httpx/client_test.go (hunk 1 above)
Comment (sp-daniel-ramos) [reply httpx/client_test.go]: done

**Pull request merged**